python/ray/experimental/gpu_buffer/torch_test.py

```diff
@@ -0,0 +1,125 @@
+import argparse
+import time
+
+import torch
+import torchvision
+import torchvision.transforms as transforms
+import torch.optim as optim
+import torch.nn as nn
+
+from models import SimpleCNN
+
+def prepare_imagenet(imagenet_path, imgsize = 256):
+    transform = transforms.Compose([
+        transforms.ToTensor(),
+        transforms.Resize((imgsize, imgsize)),
+    ])
+    trainset = torchvision.datasets.ImageFolder(root=imagenet_path, transform=transform)
+    return trainset
+
+def download_dataset(imgsize):
+    transform = transforms.Compose([
+        transforms.ToTensor(),
+        transforms.Resize((imgsize, imgsize)),
+    ])
+    trainset = torchvision.datasets.CIFAR10(root='./data', train=True, download=True, transform=transform)
+    testset = torchvision.datasets.CIFAR10(root='./data', train=False, download=True, transform=transform)
+    return trainset, testset
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Pytorch CUDA benchmark. Loads CIFAR10 and runs 3 training epochs.')
+    parser.add_argument(
+        '--pin',
+        help='Pins memory. Default is false.',
+        action='store_true',
+        required=False
+    )
+    parser.add_argument(
+        '--model',
+        help='Which model to train. Small is simple CNN, large is VGG11.',
+        choices=['small', 'large'],
+        default='large',
+        required=False,
+    )
+    parser.add_argument(
+        '--size',
+        help='Width/height to scale the images to. Only applies to CIFAR10 dataset. Prefer a power of 2.',
```

Comment:
only working for one dataset seems a little non-intuitive. Allow each dataset to have its default size and make this param optional?